crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -478,6 +478,39 @@ MAP
 .............
 ENDMAP
 
+NAME:   hellmonk_lair_entry_icy
+TAGS:   lair_entry transparent
+ORIENT: float
+MONS:   ice beast, wolf / polar bear w:3, rime drake
+NSUBST: 3 = 1:3 / * = 2.., . = 1:2 / 4 = 2... / *:.
```

Comment:
Usual suggestion here of reduced NSUBST form
```
NSUBST: 3 = 3 / 2.., . = 2 / 4=2... / .
```
and the one two lines below can have the `*` term simplified as well.